ci.sh

```diff
@@ -1,7 +1,10 @@
+# Exis on error
+set -ex
+
 export RUST_BACKTRACE=1
 export CARGO_INCREMENTAL=0
-cargo build -p lalrpop
-cargo test --all --all-features
+cargo build --bin lalrpop --features pico-args
+cargo hack test --workspace --each-feature
```

Comment:
Do the tests actually need to be run, or is `cargo hack check` sufficient?

Reply:
Cargo check merely verifies that it could possibly build.
Running the tests captures a lot more potential breakages (see the history of this PR to see all check that would've passed but actually fail when the tests is run).

Reply:
Yeah, I've tried this out and your right.